jobs/config.json

```diff
@@ -9886,7 +9886,10 @@
   "pull-kubernetes-multicluster-ingress-test": {
     "args": [
       "make",
-      "test"
+      "-j",
```

Comment:
Can you please add a comment for what this "-j", "2" does and why is it required?

Reply:
you can't comment in JSON :-)
However, the intent here is to run test and coveralls at the same time so that the presubmit doesn't take 2 * \<time to run go tests\>
https://www.gnu.org/software/make/manual/html_node/Parallel.html

Reply:
I added a comment to the commit (since we can't comment json :( )

Reply:
xref #4794

Reply:
aah yes. Thanks for the explanation!